polygeist/tools/cgeist/Test/Verification/sycl/kernels_funcs.cpp

```diff
@@ -7,7 +7,7 @@
 //
 // CHECK-MLIR-DAG:  gpu.func @_ZTS8kernel_1
 // CHECK-MLIR-SAME: kernel attributes {llvm.cconv = #llvm.cconv<spir_kernelcc>, llvm.linkage = #llvm.linkage<weak_odr>,
-// CHECK-MLIR-SAME: [[PASSTHROUGH:passthrough = \["convergent", "mustprogress", "noinline", "norecurse", "nounwind", "optnone", \["frame-pointer", "all"\], \["no-trapping-math", "true"\], \["stack-protector-buffer-size", "8"\], \["sycl-module-id", ".*/polygeist/tools/cgeist/Test/Verification/sycl/kernels_funcs.cpp"\], \["uniform-work-group-size", "true"\]\]]]} {
+// CHECK-MLIR-SAME: [[PASSTHROUGH:passthrough = \["convergent", "mustprogress", "noinline", "norecurse", "nounwind", "optnone", \["frame-pointer", "all"\], \["no-trapping-math", "true"\], \["stack-protector-buffer-size", "8"\], \["sycl-module-id", ".*/polygeist/tools/cgeist/Test/Verification/sycl/kernels_funcs.cpp"\], \["uniform-work-group-size", "true"\]\], spirv.entry_point_abi = #spirv.entry_point_abi<>]]} {
```

Comment:
This will be needed to lower to the `spirv` dialect.

Reply:
When we have a `sycl.func` or similar operation to represent kernels, we may want to drop this attribute and add when lowering to `gpu.func` kernel.